src/managers/quorum.py

```diff
@@ -198,18 +198,25 @@ def update_acls(self, event: RelationEvent | None = None) -> None:
             if not client.database:
                 continue
 
-            generated_acl = make_acl(
-                scheme="sasl",
-                credential=client.username,
-                read="r" in client.extra_user_roles,
-                write="w" in client.extra_user_roles,
-                create="c" in client.extra_user_roles,
-                delete="d" in client.extra_user_roles,
-                admin="a" in client.extra_user_roles,
+            acls = {
+                "read": "r" in client.extra_user_roles,
+                "write": "w" in client.extra_user_roles,
+                "create": "c" in client.extra_user_roles,
+                "delete": "d" in client.extra_user_roles,
+                "admin": "a" in client.extra_user_roles,
+            }
+
+            sasl_acl = make_acl(scheme="sasl", credential=client.username, **acls)
+            digest_acl = make_acl(
+                scheme="digest",
+                credential=make_digest_acl_credential(client.username, client.password),
+                **acls,
             )
-            logger.info(f"{generated_acl=}")
+            logger.info(f"{sasl_acl=}")
+            logger.info(f"{digest_acl=}")
```

Comment:
Guys, do you also think we should probably remove this log because this might log the hashed password on the Juju logs?

Reply:
I agree probably it is better to remove this log line

Reply:
I've removed both of the log lines from here.